### PR TITLE
Student+ A/O authentication system v1 done

### PR DESCRIPTION
**Pull request by Rijon1810**

### CODE REVIEW

File: models/DbModel.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const DB_Define = require("../utils/DB_Define");
-const Define = require("../utils/Define");
+const Db_Define = require("../utils/Define");
```
Comment (milon27) [inline models/DbModel.js]: const Db_Define = require("../utils/Define");
replace this with this line
const Define = require("../utils/Define"); 

@Rijon1810

**Review by milon27 — CHANGES_REQUESTED** [review]
update this DbModel.js

**Review by Rijon1810 — COMMENTED** [review]
All ok

**Comment by milon27** [comment]
good bro

**Pull request merged**